.npmignore

```diff
@@ -1,6 +1,8 @@
 webpack.*
 .eslintrc.js
-builds/
+builds/*
+builds/.*
```

Comment:
I'm not sure it is useful

Reply:
it is useful, because we want expose just `latest.js` file in npm, not other versions. If we delete this line npm don'snt have access at `latest.js` file

Reply:
oh no, I made a mistake, we can delete this line